lib/recurly/webhook.rb

```diff
@@ -37,6 +37,7 @@ module Webhook
     autoload :SubscriptionResumedNotification,      'recurly/webhook/subscription_resumed_notification'
     autoload :CanceledSubscriptionNotification,     'recurly/webhook/canceled_subscription_notification'
     autoload :ScheduledSubscriptionPauseNotification,'recurly/webhook/scheduled_subscription_pause_notification'
+    autoload :ScheduledSubscriptionUpdateNotification, 'recurly/webhook/scheduled_subscription_update_notification'
```

Comment:
Super nit and feel free to ignore: move indentation over so all the `recurly/webhook/` paths line up.

Reply:
I went round about this one. It makes PR diff nastier, but since you know what's up, I'll push as another commit on here.